eventuals/task.h

```diff
@@ -383,6 +391,8 @@ struct _TaskFromToWith final {
               std::is_convertible<Value, To_>>,
           "eventual result type can not be converted into type of 'Task'");
 
+      heap_size_ = Bytes(sizeof(HeapTask<E, From_, To_>(f(args...))));
```

Comment:
```suggestion
      heap_size_ = Bytes(sizeof(HeapTask<E, From_, To_>));
```
Here and anywhere else you need to do this.